packages/api-reference/src/components/Content/Operation/RequestBody.vue

```diff
@@ -20,6 +24,12 @@ if (prop.requestBody?.content) {
     selectedContentType.value = contentTypes.value[0] as ContentType
   }
 }
+
+const schema = computed(() => {
+  return normalizeSchema(
+    prop?.requestBody?.content?.[selectedContentType.value]?.schema,
+  )
```

Comment:
Looking at the build failures, I probably fumbled this part, because I have no experience with Vue.